### PR TITLE
MAKE-1076 Add isort to selected-tests repo

### PR DESCRIPTION
**Pull request by lydiastepanek**

### CODE REVIEW

File: .isort.cfg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[settings]
+force_grid_wrap=0
```
Comment (lydiastepanek) [inline .isort.cfg]: Most of these are to enforce compatibility with black: https://github.com/timothycrosley/isort/issues/694#issuecomment-564261886

File: evergreen.yml [hunk 2]
```diff
@@ -60,3 +60,14 @@ tasks:
           . venv/bin/activate
 
           pydocstyle src
+
+- name: check_isort
```
Comment (dbradf) [inline evergreen.yml]: We might want to consider using the [pytest plugin ](https://pypi.org/project/pytest-isort/)

File: src/selectedtests/task_mappings/create_task_mappings.py [hunk 3]
```diff
@@ -1,20 +1,19 @@
...
 from evergreen.manifest import ManifestModule
-from git import Repo, DiffIndex
-from structlog import get_logger
-from tempfile import TemporaryDirectory
-
+from git import DiffIndex, Repo
 from selectedtests.evergreen_helper import get_evg_project
 from selectedtests.git_helper import get_changed_files, init_repo
 from selectedtests.task_mappings.version_limit import VersionLimit
+from structlog import get_logger
```
Comment (dbradf) [inline src/selectedtests/task_mappings/create_task_mappings.py]: It looks like `isort` is treating the local library as a 3rd party library. Do you know if there is a way to tell it the difference?

Comment (lydiastepanek) [reply src/selectedtests/task_mappings/create_task_mappings.py]: Adding the `known_first_party` config option seems to have solved this

**Comment by lydiastepanek** [comment]
evergreen merge

**Pull request merged**